lib/cloud_controller/dependency_locator.rb

```diff
@@ -59,6 +59,18 @@ def runners
       @dependencies[:runners] || register(:runners, VCAP::CloudController::Runners.new(config))
     end
 
+    def periodic_updater
+      @dependencies[:periodic_updater] ||
+        register(:periodic_updater,
+                 VCAP::CloudController::Metrics::PeriodicUpdater.new(
+                   Time.now.utc,
+                   Steno::Sink::Counter.new,
+                   Steno.logger('cc.api'),
+                   VCAP::CloudController::Metrics::StatsdUpdater.new(statsd_client),
```

Comment:
Should we also have a singleton `StatsdUpdater` like  `prometheus_updater`?

Reply:
Maybe. But I would not add this within this PR as it is quite big already and not required.